test/routers/user.test.js

```diff
@@ -0,0 +1,109 @@
+const request = require('supertest');
+const express = require('express');
+const router = require('../../routes/users');
+const ApiError = require('../../utils/api-error');
+const { users } = require('../../models');
+require('dotenv').config();
```

Comment:
don't import .env in your unit test. the file can't import in server CI-test. if you import, you should have a test version env in test.